src/content/legal/terms-and-conditions.md

```
@@ -0,0 +1,61 @@
+---
+title: Terms and Conditions
+description: Read the Terms and Conditions governing the use of this service.
+lastUpdated: 2026-04-02
+---
+
+Please read these Terms and Conditions carefully before using **https://rafay99.com** (the "Website").
+
+By accessing or using this Website, you agree to be bound by these Terms. If you do not agree, please do not use the Website.
+
+### 1. Who I Am
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)